projects/packages/my-jetpack/webpack.config.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+const getBaseWebpackConfig = require( '@automattic/calypso-build/webpack.config.js' );
+const DependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
+const path = require( 'path' );
+
+const isDevelopment = process.env.NODE_ENV !== 'production';
+
+const baseConfig = getBaseWebpackConfig(
+	{ WP: false },
+	{
+		entry: {}, // We'll override later
+		'output-filename': '[name].js',
+		'output-path': path.join( __dirname, './build' ),
+	}
+);
+
+module.exports = [
+	{
+		...baseConfig,
+		resolve: {
+			...baseConfig.resolve,
+			modules: [ 'node_modules' ],
+		},
+		devtool: isDevelopment ? 'source-map' : false,
+		entry: { index: path.join( __dirname, './_inc/admin.jsx' ) },
+		plugins: [
+			...baseConfig.plugins,
+			new DependencyExtractionWebpackPlugin( { injectPolyfill: true } ),
+		],
+	},
+];
```

Comment:
This file needs to be updated along the lines of #21580. I think something like this:
```suggestion
/**
 * External dependencies
 */
const jetpackWebpackConfig = require( '@automattic/jetpack-webpack-config/webpack' );
const path = require( 'path' );

module.exports = [
	{
		entry: {
			index: './_inc/admin.jsx',
		},
		mode: jetpackWebpackConfig.mode,
		devtool: jetpackWebpackConfig.isDevelopment ? 'source-map' : false,
		output: {
			...jetpackWebpackConfig.output,
			filename: '[name].js',
			path: path.resolve( './build' ),
		},
		optimization: {
			...jetpackWebpackConfig.optimization,
		},
		resolve: {
			...jetpackWebpackConfig.resolve,
		},
		node: false,
		plugins: [
			...jetpackWebpackConfig.StandardPlugins( {
				DependencyExtractionPlugin: { injectPolyfill: true },
			} ),
		],
		module: {
			strictExportPresence: true,
			rules: [
				// Transpile JavaScript
				jetpackWebpackConfig.TranspileRule( {
					exclude: /node_modules\//,
				} ),

				// Transpile @automattic/jetpack-* in node_modules too.
				jetpackWebpackConfig.TranspileRule( {
					includeNodeModules: [ '@automattic/jetpack-' ],
				} ),

				// Handle CSS.
				{
					test: /\.(?:css|s[ac]ss)$/,
					use: [
						jetpackWebpackConfig.MiniCssExtractLoader(),
						jetpackWebpackConfig.CssCacheLoader(),
						jetpackWebpackConfig.CssLoader( {
							importLoaders: 1, // Set to the number of loaders after this one in the array, e.g. 2 if you use both postcss-loader and sass-loader.
						} ),
						'sass-loader',
					],
				},

				// Handle images.
				jetpackWebpackConfig.FileRule(),
			],
		},
	},
];
```

Reply:
Do we need to keep `mode`, `optimization` and `resolve` if we are not changing anything on them?

Reply:
Unlike the old calypso-build setup, the new setup doesn't provide a "base" config to extend. It only supplies the pieces, like `mode` and `optimization` and `resolve`.

I could be talked into changing that, but I kind of like having to be a little more intentional about what exactly gets used.